phpunit.xml.dist

```diff
@@ -16,6 +16,7 @@
 	</php>
 	<testsuites>
 		<testsuite name="WP_Tests">
+			<directory>./tests/</directory>
```

Comment:
This seems like an odd entry, considering the one below it as well.

Reply:
The [default value of `suffix`](https://github.com/sebastianbergmann/phpunit/blob/master/phpunit.xsd#L134) is `Test.php`, so the new line is definitely needed.

The actual issue would be the lack of removal of the line below (with `suffix="-test.php"`).

I think I left it in because the `recommended-api-test.php` test file never got renamed into `*Test.php` because Jeff was still working on it via another big PR, so I didn’t want to break that.

Not a blocker for 2.5.2 release, but a good reminder to see if the `recommended-api-test.php`-related PR (https://github.com/Parsely/wp-parsely/pull/318?) got merged, then the second `directory` element can be removed from the config.

Reply:
Opened #379 to address.